payment-handler/idlharness.https.any.js

```diff
@@ -0,0 +1,20 @@
+// META: script=/resources/WebIDLParser.js
+// META: script=/resources/idlharness.js
+
+'use strict';
+
+// https://w3c.github.io/payment-handler/
+
+promise_test(async () => {
+  const idl = await fetch('/interfaces/payment-handler.idl').then(r => r.text());
```

Comment:
How about:

```JS
const [idl, sw, dw, dom] = Promise.all([
  fetch("/interfaces/ServiceWorker.idl").then(r => r.text()),
  fetch("/interfaces/dedicated-workers.idl").then(r => r.text()),
  fetch("/interfaces/payment-handler.idl").then(r => r.text()),
  fetch("/interfaces/dom.idl").then(r => r.text()),
]);
```

Reply:
Or, probably better (just trying to avoid downloading in sequence):

```JS
const idls = [
  "/interfaces/ServiceWorker.idl",
  "/interfaces/dedicated-workers.idl",
  "/interfaces/payment-handler.idl",
  "/interfaces/dom.idl",
];
const [idl, sw, dw, dom] = Promise.all(
  idls.map(url => fetch(url).then(r => r.text()))
);
```

Reply:
Done (the pattern you see in the update I've actually used all over the show for other tests, not sure why I dropped the ball here).